### DISCUSSION TITLE
no workaround for multiplayer?

### DISCUSSION THREAD

**Discussion by snackmasta** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#31] [General]
even for just 2-3 player or some kind of LAN thingy?

**Comment by cwmonkey** [comment]
The problem is if your client has access to the HG servers when the game starts it will override your .json file. There are some theoretical ways around this:

1. A mod could potentially block the call to the HG servers for an updated json file. I'm not sure if anyone has looked into this.
2. The call to the HG server could potentially be blocked/intercepted with some sort of firewall/vpn override etc

I just tried elevating the permissions on the .json file so the game couldn't delete/override it, but that doesn't work (the json file stayed, but the game still gave me the newest expedition), so it's definitely the call to the HG servers that needs to be short circuited.
